cli/src/commands/developer/execute.rs

```diff
@@ -109,7 +110,7 @@ impl Execute {
     /// Construct and process the execution transaction.
     fn construct_execution<N: Network>(&self) -> Result<String> {
         // Specify the query
-        let query = Query::from(&self.query);
+        let query = Query::<N, BlockMemory<N>>::from(&self.query);
```

Comment:
@mikebenfield when compiling Leo with the latest snarkVM staging, do you also end up having to specify `BlockMemory`, or do you have another workaround? No rush.